main.js

```diff
@@ -362,42 +412,116 @@ client.on('message', async (msg) => {
       message = 'Please enter a valid stat type (verified|major|year|transfer|affiliation)';
     }
   }
-
-  else if (command === 'help') {
+  else if(command === 'help' && isModOrAdmin(member)) {
```

Comment:
I don't think there should be an `isModOrAdmin(member)` here, since that makes help not work unless you're a mod or admin!